kiwi/boot/image/__init__.py

```diff
@@ -28,14 +28,16 @@ class BootImage(object):
     """
         BootImge factory
     """
-    def __new__(self, xml_state, target_dir, root_dir=None):
+    def __new__(self, xml_state, target_dir, root_dir=None, signing_keys=None):
         initrd_system = xml_state.build_type.get_initrd_system()
         if not initrd_system:
             initrd_system = 'kiwi'
         if initrd_system == 'kiwi':
-            return BootImageKiwi(xml_state, target_dir)
+            return BootImageKiwi(
+                xml_state, target_dir, signing_keys=signing_keys
```

Comment:
We need the key=value style here, since base init signatures is 
 ```python
def __init__(self, xml_state, target_dir, root_dir=None, signing_keys=None)
```
Then I need the key=value style to state the third parameter is `signing_keys` instead of `root_dir`